src/components/Tooltip/__tests__/Tooltip.spec.js

```diff
@@ -40,8 +41,7 @@ describe('Tooltip', () => {
     const trigger = tooltip.find('button').last()
     trigger.last().simulate('click')
 
-    const tree = renderer.create(tooltip).toJSON()
-    expect(tree).toMatchSnapshot()
+    expect(tooltip).toMatchSnapshot()
```

Comment:
we should add some validation that the overlay actually appears. a snapshot doesn't provide that. it's not clear that the overlay is actually present in the dom

Reply:
Made the tests more specific.